### PR TITLE
Feature/opmon

### PR DESCRIPTION
**Pull request by mroda88**

### CODE REVIEW

File: schema/datahandlinglibs/opmon/datahandling_info.proto [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  uint64 num_tps_suppressed_too_long = 4; // Number of TPs that were discarded because their time over threshold was too large
+}
+
+message TPChannelInfo {
+  uint64 channel_id = 1; //Offline channel identifier
+  uint64 number_of_tps = 2; // Numer of hits, i.e. TPs
+}
+
+message TPChannelsInfo {
+  repeated TPChannelInfo channels = 1;
```
Comment (mroda88) [inline schema/datahandlinglibs/opmon/datahandling_info.proto:74]: Repeated fields cannot be used for monitoring

File: schema/datahandlinglibs/opmon/datahandling_info.proto [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+message FixedRateDataProcessorInfo {
+  uint64 num_seq_id_errors = 1;
+  uint64 min_seq_id_jump = 2;
+  uint64 max_seq_id_jump= 3;
+  uint64 num_ts_errors= 4;
+}
+
+message HitFindingInfo {
+  float rate_tp_hits = 1; // TP hit rate in kHz
```
Comment (mroda88) [inline schema/datahandlinglibs/opmon/datahandling_info.proto:62]: We can use double here

File: schema/datahandlinglibs/opmon/datahandling_info.proto [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  uint64 max_request_response_time = 24; // Max response time in us for the requests handled in between publication calls
+  uint64 num_buffer_cleanups = 31; // Number of latency buffer cleanups
+  uint64 num_periodic_sent = 41; // Number of periodic sends
+  uint64 num_periodic_send_failed = 42; // Number of failed periodic sends
+}
+
+message RecordingInfo {
+  string recording_status = 1; // Recording status
+  uint64 packets_recorded = 2; // Number of packets processed
+  float bytes_recorded = 3; // Throughput of processed packets
```
Comment (mroda88) [inline schema/datahandlinglibs/opmon/datahandling_info.proto]: Double

Comment (glehmannmiotto) [reply schema/datahandlinglibs/opmon/datahandling_info.proto]: changed

File: schema/datahandlinglibs/opmon/datahandling_info.proto (hunk 2 above)
Comment (glehmannmiotto) [reply schema/datahandlinglibs/opmon/datahandling_info.proto:62]: ok

**Review by glehmannmiotto — APPROVED** [review]
OK; the repeated protobuf field is to be sorted out, still

**Pull request merged**